MovieApp/PresentationLayer/MovieDetails/MovieDetailsViewController+Design.swift

```diff
@@ -0,0 +1,38 @@
+import SnapKit
+
+extension MovieDetailsViewController: ConstructViewsProtocol {
+
+    func buildViews() {
+        createViews()
+        styleViews()
+        defineLayoutForViews()
+    }
+
+    func createViews() {
+        navigationView = NavBarView()
+        view.addSubview(navigationView)
+
+        headerView = MovieHeaderView()
+        view.addSubview(headerView)
+    }
+
+    func styleViews() {
+        view.backgroundColor = .white
+
+        navigationView.isBackButtonHidden = false
+    }
+
+    func defineLayoutForViews() {
+        navigationView.snp.makeConstraints {
+            $0.top.leading.trailing.equalToSuperview()
+            $0.height.equalTo(NavBarView.defaultHeight)
+        }
+
+        headerView.snp.makeConstraints {
+            $0.top.equalTo(navigationView.snp.bottom)
+            $0.leading.trailing.equalToSuperview()
+            $0.height.equalTo(headerView.snp.width).multipliedBy(0.9)
```

Comment:
This could look weird on larger screens. I would rather give it a fixed height. But you can leave it for now and we will revisit in the future.